src/Microsoft.Health.Fhir.Core/Features/KnownQueryParameterNames.cs

```diff
@@ -10,44 +10,48 @@ namespace Microsoft.Health.Fhir.Core.Features
     /// </summary>
     public static class KnownQueryParameterNames
     {
-        /// <summary>
-        /// The format query parameter.
-        /// </summary>
-        public const string Format = "_format";
-
-        /// <summary>
-        /// The pretty query parameter.
-        /// </summary>
-        /// <remarks>True if the client wishes to request for pretty-printed resources (either in JSON or XML), false otherwise.</remarks>
-        public const string Pretty = "_pretty";
-
         public const string At = "_at";
 
+        public const string Before = "_before";
+
         /// <summary>
         /// The continuation token parameter.
         /// </summary>
         public const string ContinuationToken = "ct";
 
         public const string Count = "_count";
 
-        public const string Since = "_since";
+        public const string DestinationConnectionSettings = "_destinationConnectionSettings";
+
+        public const string DestinationType = "_destinationType";
+
+        /// <summary>
+        /// The format query parameter.
+        /// </summary>
+        public const string Format = "_format";
 
         public const string LastUpdated = "_lastUpdated";
 
-        public const string Before = "_before";
+        public const string Mode = "mode";
 
-        public const string DestinationType = "_destinationType";
+        /// <summary>
+        /// The pretty query parameter.
+        /// </summary>
+        /// <remarks>True if the client wishes to request for pretty-printed resources (either in JSON or XML), false otherwise.</remarks>
+        public const string Pretty = "_pretty";
 
-        public const string DestinationConnectionSettings = "_destinationConnectionSettings";
+        public const string Profile = "profile";
```

Comment:
Unfortunately this is also a specified query parameter: 
https://www.hl7.org/fhir/resource-operation-validate.html#examples

Reply:
Yea, that's why I suggested naming (the variable name) it something specific for validation since it would probably make more sense to have the more broad variable be `Profile = "_profile"`.